specs/dataplaneapi/dataplaneapi.spec

```diff
@@ -0,0 +1,144 @@
+################################################################################
+
+%global crc_check pushd ../SOURCES ; sha512sum -c %{SOURCE100} ; popd
```

Comment:
Macro for [automatic CRC generation and check](https://github.com/essentialkaos/rpmbuilder/wiki/Automatic-SHA-512-checksum-generation) defined but there is no `Source100` or macro usage.